src/App.js

```diff
@@ -0,0 +1,21 @@
+import "./styles.css";
+import { Navbar, Home, LikedVideos,WatchLater,WatchHistory,Playlists} from "./Components";
+import { VideoPlayer } from "./Components";
```

Comment:
You can put VideoPlayer along with Navbar, Home, LikedVideos,WatchLater,WatchHistory,Playlists as all are imported from ./Components;